src/Microsoft.SqlTools.ServiceLayer/SqlContext/QueryExecutionSettings.cs

```diff
@@ -26,6 +29,16 @@ public class QueryExecutionSettings
         /// </summary>
         private const int DefaultMaxXmlCharsToStore = 2097152; // 2 MB - QE default
 
+        /// <summary>
+        /// Default selection of returning an actual XML showplan with all batches
+        /// Do not return any execution plan by default
+        /// </summary>
+        private ExecutionPlanOptions DefaultExecutionPlanOptions = new ExecutionPlanOptions() 
```

Comment:
(I'm asking more of an existential question than directly asking about these specific settings)